tools/contrib/run-clang-format.py

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env python
```

Comment:
You're going to need a `THIRD-PARTY-NOTICES` file as in HiveMind if you use this script

Reply:
Done